src/Library/Configs/IMH_Central.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * BoldGrid Library Configs IMH Central Class
```

Comment:
Can we add a comment with a very brief explanation about what this class is used for and why?